qa-physics/qaCut.groovy

```diff
@@ -457,10 +457,13 @@ inList.each { obj ->
       totN = listN.sum()
       totF = listF.sum()
       totU = listU.sum()
-      totA = totN / totF
+      totA = totF > 1e-3 ? totN / totF : 0 
```

Comment:
Why `1e-3` and not `0`? Similarly for `totLT` below.

Reply:
Should be 0, yes.